packages/react/presence/src/Presence.stories.tsx

```diff
@@ -28,6 +28,35 @@ export const WithMultipleOpenAndCloseAnimations = () => (
   <Animation className={multipleOpenAndCloseAnimationsClass} />
 );
 
+export const WithDeferredMountAnimation = () => {
+  const ref = React.useRef<HTMLDivElement>(null);
+  const timerRef = React.useRef(0);
+  const [open, setOpen] = React.useState(false);
+  const [animate, setAnimate] = React.useState(false);
+
+  React.useEffect(() => {
+    if (open) {
+      timerRef.current = window.setTimeout(() => {
+        setAnimate(true);
+      }, 150);
+    } else {
+      setAnimate(false);
+      window.clearTimeout(timerRef.current);
+    }
+  }, [open]);
+
+  return (
+    <>
+      <Toggles nodeRef={ref} open={open} onOpenChange={setOpen} />
+      <Presence present={open}>
+        <div className={animate ? mountAnimationClass : noAnimationClass} ref={ref}>
```

Comment:
I found the flicker where it appears and then animates in jarring. I thought it was a bug in the story. Thoughts on changing to this so it looks more like a real use-case?

```suggestion
        <div hidden={!animate} className={animate ? mountAnimationClass : undefined} ref={ref}>
```

Reply:
Oh yeah this is better, I did wonder if this looked a bit broken without prior knowledge

Reply:
@jjenzz I realised using `hidden` would mask the issue we want check for. I've added a description instead https://github.com/radix-ui/primitives/pull/851/commits/9b0fb0e131da7b5913bd688e363ceb43c32bbacf let me know if you think this is enough.

Reply:
That works :+1: thanks